include/veriblock/blockchain/pop/fork_resolution.hpp

```diff
@@ -506,7 +510,7 @@ struct PopAwareForkResolutionComparator {
     {
       auto guard = ing_->deferForkResolutionGuard();
 
-      if (!sm.apply(*chainB.first(), *chainB.tip(), state, false)) {
+      if (!sm.apply(*chainB.first(), *chainB.tip(), state)) {
```

Comment:
why did you change this? 
this apply applies chain B together with chain A, and this `false` indicated that we should not set flag `CAN_BE_APPLIED` during this state change